### ISSUE TITLE
Only one Provider & No Streams

### ISSUE THREAD

**Issue by BeckerC54** [Tempest0580/tempest#103]
I’m on Kodi 18.8 Leia and I’ve been using Tempest successfully right up until last night. For example, I’d been watching Season 19 of Hell’s Kitchen when it started to freeze while searching for links for the episode. I saw advice to deselect VIDNEXT, but it didn’t help. I followed some other advice to clear the cache and clear providers and perhaps that was an error because now it’s worse. Now when I search for links for an episode (any episode, any TV show), it returns only **ONE** provider: FSAPI | CDN, which when chosen always returns no streams available. Just a day ago I’d get tons of links for the same episodes and now it’s just this one each time. What can I do to fix it?

**Comment by yugioh9** [comment]
yes iam haveing the same problem

**Comment by tonpoes** [comment]
Indeed, same here. I'am running on Kodi 18.6. Normaly using Tempest a lot; but now only 1 or 2 FSAPI links appear. Mostly no stream available..... Also all my other video addons  with the same problem. Help please

**Comment by chasparr2** [comment]
The same just happened to me within the last week, Tempest was working fine. Now most (?) of it I get only 1 provider also, fsapi cdn mp4 with no stream. I checked to make sure all torrents was enabled. I'm using kodi 18.7 with IPVanish vpn and real-debrid. I did a quick check on others, DejaVu, Marauder, Base, The Crew etc and don't see the same problem though it was a quick scan. So, anybody, any ideas? Is this a tempest thing now?

**Comment by DJSleeves** [comment]
Me as well. Originally I thought my ISP blocked me.  I'm glad to see you guys are having the same issue.  So I know its not my ISP.

**Comment by dolittle26** [comment]
Hi I am having problems on kodi.  Keeps going to only 1 provider fsapi and then says no stream available  
I've uninstalled and reinstalled kodi and crew 19.4 add on but still cant make it work. Please help
